xc/xc7/fasm2bels/verilog_modeling.py

```diff
@@ -1604,9 +1600,9 @@ def output_nets(self):
 
         for net_wire_pkey, net in self.nets.items():
             if net_wire_pkey == ZERO_NET:
-                yield 'set net [get_nets {<const0>}]'
```

Comment:
You broke the constant net logic, this will not work.

Reply:
Fixed